api/config/config_local.json

```diff
@@ -100,7 +100,6 @@
     "enableV3DataUserCodeOfConduct": true,
     "enableEventDateModifier": false,
     "enableResearchPurposePrompt": true,
-    "enableCohortBuilderV2": true,
```

Comment:
Per the [feature flags doc](https://docs.google.com/document/d/133JzAcrIf_ZKFqzL2jvVaBbxb69NMMSjs7iTuj8xW0A/edit#heading=h.aqdwmiwuhrsr), the safe method for flag removal is to removal all usage/code in one release (99% of this PR), then remove the configs (JSON and ServerConfigController) in the following release, as a separate PR. Otherwise there can be complications during the release on account of the configs, API code, and UI code not being exactly synchronized for a period, per a recent post-mortem.

Reply:
Thanks @calbach, I'll revert the flags.